jwst/datamodels/schemas/guider_fgs_cal.schema.yaml

```diff
@@ -0,0 +1,132 @@
+allOf:
```

Comment:
Do we need "fgs" in these names? Why not just guider_cal.schema.yaml and guider_raw.schema.yaml?